server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,12 @@
+import Serializer from "./Serializer.js";
+
+class ReviewSerialzer extends Serializer {
+  static async getDetail(review) {
+    const user = await review.$relatedQuery("user")
+    review.firstName = user.firstName
+    review.lastName = user.lastName
+    return this.serialize(review, ["id", "title", "rating", "text", "pizzaId", "userId", "firstName", "lastName"])
```

Comment:
I'd recommend separating concerns here
- first serialize the review itself
- query the related user
- then attach just the first and last name of the user to the serialized review, just like you have on lines 6-7

it's a little "muddied" by sending additional properties with the review to be serialized